File: src/main/java/com/adamkoch/iptables/matches/IgnoreCaseExtensionMatchOption.java
package com.adamkoch.iptables.matches;

/**
 * @author aakoch
 * @since 0.1.0
 */
public class IgnoreCaseExtensionMatchOption extends GenericExtensionMatchOption {

  public static final IgnoreCaseExtensionMatchOption DEFAULT = new IgnoreCaseExtensionMatchOption();

  private IgnoreCaseExtensionMatchOption() {
  }

  @Override
  public String asString() {
    return "--icase";
  }
}

File: src/main/java/com/adamkoch/iptables/matches/IgnoreCaseExtensionMatchOption.kt
package com.adamkoch.iptables.matches

/**
 * @author aakoch
 * @since 0.1.0
 */
class IgnoreCaseExtensionMatchOption private constructor() :
    GenericExtensionMatchOption("--icase") {
    override fun asString(): String {
        return "--icase"
    }

    companion object {
        val DEFAULT = IgnoreCaseExtensionMatchOption()
    }
}